insights/collect.py

```diff
@@ -251,6 +251,9 @@
 
         - name: insights.components.cryptsetup
           enabled: true
+
+        - name: insights.combiners.cryptsetup.luks1_block_devices
```

Comment:
If It's okay to run the `luksmeta` against all the `by-uuid` devices, this component is not required and is not needed to be `pre-loaded`.